package.json

```diff
@@ -13,16 +13,34 @@
     "lint:js": "node ./node_modules/eslint/bin/eslint.js .",
     "lint:js-passing": "node ./node_modules/eslint/bin/eslint.js --quiet --config=.eslintrc.passing.json .",
     "lint:js-stats": "node ./node_modules/eslint/bin/eslint.js --format=./scripts/js/eslint-stats-by-type.js .",
-    "lint:css": "stylelint \"**/*.css\"",
-    "lint:css-checkstyle": "stylelint \"**/*.css\" --custom-formatter ./node_modules/stylelint-checkstyle-formatter/index.js"
+    "lint:css": "stylelint \"css/dist/**/*.css\"",
```

Comment:
Why are we linting the compiled CSS files?

Reply:
Changed to `css/src/**/*.css`.